src/main/features/oak/persistence/oak_persistence_mongods.json

```diff
@@ -13,7 +16,7 @@
     "configurations":{
         "org.apache.jackrabbit.oak.plugins.document.DocumentNodeStoreService":{
             "db":"sling",
-            "mongouri":"mongodb://localhost:27017"
+            "mongouri":"mongodb://localhost:${mongo.port}"
```

Comment:
Maybe add the [Felix ConfigAdmin interpolation plugin](https://github.com/apache/felix-dev/tree/master/configadmin-plugins/interpolation)?

We could then do something like

```
  "mongouri": "mongodb://localhost:$[env:MONGO_PORT;default=27017]"
````

Reply:
This would have meant adding a dependency to that plugin for Sling starter users, right? If the syntax is used in a feature model file that we distribute.

Anyway, I have added support for feature model variables to the `feature-launcher-maven-plugin` in https://issues.apache.org/jira/browse/SLING-10420 and pushed changes which use that. The mongo port is now dynamic.